models/vista3d/scripts/evaluator.py

```diff
@@ -243,8 +243,8 @@ def _iteration(self, engine: SupervisedEvaluator, batchdata: dict[str, torch.Ten
                 label_set = np.arange(output_classes).tolist()
             label_prompt = torch.tensor(label_set).to(engine.state.device).unsqueeze(-1)
             # point prompt is generated withing vista3d,provide empty points
-            points = torch.zeros(label_prompt.shape[0], 1, 3)
-            point_labels = -1 + torch.zeros(label_prompt.shape[0], 1)
+            points = torch.zeros(label_prompt.shape[0], 1, 3).to(inputs.device)
```

Comment:
HI @heyufan1995 , could you help to confirm if the changes here are correct? I got device mismatch error thus changed here, thanks in advance.